content/language-support/_index.md

```diff
@@ -17,3 +17,4 @@ Ruby|https://www.rubydoc.info/gems/opencensus
 Erlang|https://hexdocs.pm/opencensus/
 Python|https://census-instrumentation.github.io/opencensus-python/trace/api/index.html
 PHP|https://packagist.org/packages/opencensus/opencensus
+Node|https://github.com/census-instrumentation/opencensus-node
```

Comment:
Node.js

Reply:
@hvent90, after this PR is merged, could you please make an update to the languages pictures too?

Reply:
Sure! Updated the picture too!